src/Utilities/RecoilAtoms.res

```diff
@@ -22,7 +22,7 @@ let paymentTokenAtom = Recoil.atom(
 let showCardFieldsAtom = Recoil.atom("showCardFields", false)
 let phoneJson = Recoil.atom("phoneJson", Loading)
 let cardBrand = Recoil.atom("cardBrand", "")
-let payNowButtonDisable = Recoil.atom("payNowButtonDisable", true)
+let payNowButtonDisable = Recoil.atom("payNowButtonDisable", false)
```

Comment:
Why a RecoilAtom can it be done as a local state as it's only required in PayNowButton file.

Reply:
pushed the required changes